src/cardboard-vr-display.js

```diff
@@ -149,6 +149,11 @@ CardboardVRDisplay.prototype.beginPresent_ = function() {
       e.preventDefault();
     }.bind(this), function(e) {
       // Back clicked.
+
+      // In case of a custom calllback defined by the user, trigger it
+      if (typeof WebVRConfig.BACKACTION_CALLBACK === 'function') {
+        WebVRConfig.BACKACTION_CALLBACK();
```

Comment:
should this be called before or after `exitPresent`. personally, as a dev, I'd want to call it after the `vrDisplay.exitPresent()` `Promise` resolves or rejects (or get a `Promise` back).